File: src/main/java/com/github/rogerbarton/intellijsv/psi/SvElementType.java
package com.github.rogerbarton.intellijsv.psi;

import com.github.rogerbarton.intellijsv.SvLanguage;
import com.intellij.psi.tree.IElementType;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class SvElementType extends IElementType {

    public SvElementType(@NotNull @NonNls String debugName) {
        super(debugName, SvLanguage.INSTANCE);
    }
}

File: src/main/kotlin/com/github/rogerbarton/intellijsv/core/psi/SvElementType.kt
package com.github.rogerbarton.intellijsv.core.psi

import com.github.rogerbarton.intellijsv.SvLanguage
import org.jetbrains.annotations.NonNls
import com.intellij.psi.tree.IElementType

class SvElementType(debugName: String) : IElementType(debugName, SvLanguage)